vars/deploy-on-kvm.yml

```diff
@@ -1,2 +1,2 @@
 # number of caasp worker nodes
-deploy_on_kvm_caasp_workers: 1
+deploy_on_kvm_caasp_workers: 3
```

Comment:
Apparently we need 3 nodes for rook monitor quorums. Earlier I have 2 worker node and ceph monitor and manager pods never came up with that number